pkg/ccl/crosscluster/logical/dead_letter_queue_test.go

```diff
@@ -111,60 +115,145 @@ func TestDLQClient(t *testing.T) {
 
 	sqlDB := sqlutils.MakeSQLRunner(db)
 	sqlDB.Exec(t, `CREATE TABLE foo (a INT)`)
+	sqlDB.Exec(t, `CREATE SCHEMA baz`)
+	sqlDB.Exec(t, `CREATE TABLE baz.foo (a INT)`)
+	sqlDB.Exec(t, `CREATE TABLE baz.bar (a INT)`)
 
-	tableName := "foo"
-	tableDesc := desctestutils.TestingGetPublicTableDescriptor(kvDB, s.Codec(), "defaultdb", tableName)
+	tableNames := []fullyQualifiedTableName{
+		{
+			database: defaultDbName,
+			schema:   "public",
+			table:    "foo",
+		},
+		{
+			database: defaultDbName,
+			schema:   "baz",
+			table:    "foo",
+		},
+		{
+			database: defaultDbName,
+			schema:   "baz",
+			table:    "bar",
+		},
+	}
+
+	tableNameToDesc := make(map[string]catalog.TableDescriptor)
+	tableIDToName := make(map[int32]fullyQualifiedTableName)
+
+	for _, name := range tableNames {
+		desc := desctestutils.TestingGetTableDescriptor(kvDB, s.Codec(), name.database, name.schema, name.table)
+		tableIDToName[int32(desc.GetID())] = name
+		tableNameToDesc[fmt.Sprintf("%s.%s", name.schema, name.table)] = desc
+	}
+
+	// Build family desc for cdc event row
 	familyDesc := &descpb.ColumnFamilyDescriptor{
 		ID:   descpb.FamilyID(1),
 		Name: "",
 	}
 
-	ed, err := cdcevent.NewEventDescriptor(tableDesc, familyDesc, false, false, hlc.Timestamp{})
-	require.NoError(t, err)
-
-	tableID := int32(tableDesc.GetID())
-	dlqClient := InitDeadLetterQueueClient(ie)
-	require.NoError(t, dlqClient.Create(ctx, []int32{tableID}))
-
-	var tableNameQueryResult string
-	sqlDB.QueryRow(t, `SELECT table_name FROM [SHOW TABLES FROM defaultdb]`).Scan(&tableNameQueryResult)
-	require.Equal(t, tableName, tableNameQueryResult)
+	dlqClient := InitDeadLetterQueueClient(ie, tableIDToName)
+	require.NoError(t, dlqClient.Create(ctx))
```

Comment:
could you check that that the right enums were created in the right databases?

Reply:
tests are updated! :)